revolv/static/project.scss

```diff
@@ -43,6 +43,439 @@
     }
 }
 
+body {
+    font-family: Lato;
+}
+
+.row {
```

Comment:
Oh okay I guess that's alright, it will still cause problems if you have rows within rows though I think?